api/routes/api_news.py

```diff
@@ -73,3 +82,54 @@ def specific_article(service: NewsService = Provide[Container.news_service]):
         logger.error(f"Error in POST /: {str(e)}")
         get_custom_error_response(e)
 
+
+
+@news_app.route("/get/recommendation", methods=["GET"])
+@jwt_required()
+@inject
+@handle_exceptions
+@logger.log_function_call()
+async def get_recommendations_endpoint(
+        service_recs: RecommendationService = Provide[Container.recommendation_service],
+        service_news: NewsService = Provide[Container.news_service],
+    ):
+    try:
+        user_email = get_jwt_identity()
+        rec = await service_recs.get_recommendations_from_db(user_email)
+        all_recs_with_data = await service_news.send_all_info_from_blob_to_user(rec)
+
+        return all_recs_with_data
+
+    except CustomQSportException as e:
+        logger.error(f"Error in Get Recommendations /: {str(e)}")
+        get_custom_error_response(e)
+
+
+@news_app.route("/fast/recommendation", methods=["POST"])
+@inject
+@handle_exceptions
+@logger.log_function_call()
+async def fast_generate_recommendation_endpoint(service_recs: RecommendationService = Provide[Container.recommendation_service]):
```

Comment:
endpoint name is not the best either

ALSO, I don't see a reason why y'all keep adding "endpoint" in the end of the name
Seems a little redundant